docker-hub/builds/advanced.md

```diff
@@ -68,10 +68,10 @@ repository at the same directory level as your Dockerfile. Create a file called
 builder process can execute, such as `docker` and `bash` commands (prefixed
 appropriately with `#!/bin/bash`).
 
-These hooks run on an instance of [Amazon Linux 2](https://aws.amazon.com/amazon-linux-2/){:target="_blank" rel="noopener" class="_"},
-a distro based on Red Hat Enterprise Linux (RHEL), which includes interpreters
+These hooks run on an instance of [Ubuntu Bionic](https://releases.ubuntu.com/18.04/){:target="_blank" rel="noopener" class="_"},
+a distro based on Debian, which includes interpreters
```

Comment:
We can probably remove the "a distro based on Debian". For Amazon Linux that was slightly useful, but I guess most people know what Ubuntu is

Reply:
Yup. fixed that